src/detection/Bootprint.cc

```diff
@@ -158,7 +184,7 @@ class LegacyBootprintPersistenceHelper {
         spanY(spanSchema.addField<int>("y", "The row of the span", "pixel")),
         spanX0(spanSchema.addField<int>("x0", "First column of span (inclusive)", "pixel")),
         spanX1(spanSchema.addField<int>("x1", "Second column of span (inclusive)", "pixel"))
-        {}
+        {spanSchema.getCitizen().markPersistent();}
```

Comment:
This was discussed in person, but leaving a not here for future readers. This is a standard part of the persistence layer. This basically just keeps Citizen from mistakenly thinking memory is leaking as a variable is static and does not get cleared at the time the memory examiner expects it to.